### PR TITLE
Move CI to actions

### LINKED ISSUE #1069: Use github actions for ci / cd

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes #1069

### CODE REVIEW

**Comment by hewigovens** [comment]
@maxUo need your help to hook up azure deploy

**Comment by hewigovens** [comment]
I think blockatlas has too many code owners? if you assign it to everyone (~= no one) @EnoRage @vikmeup , 2~3 should be enough?

**Comment by EnoRage** [comment]
> I think blockatlas has too many code owners? if you assign it to everyone (== no one) @EnoRage @vikmeup , 2~3 should be enough?

++++

**Comment by EnoRage** [comment]
@hewigovens let's just have 2 pipelines with all tests consequently. We can lint and build before unit tests

**Comment by EnoRage** [comment]
@maxUo please, help us also

**Comment by maxUo** [comment]
So what is the end result.  Leave 2 separate Pipeline?  Or combine GA with AZP (Deploy) together?

@hewigovens @EnoRage

**Comment by kolya182** [comment]
How about adding caching ?

**Comment by hewigovens** [comment]
@kolya182 what cache do you need? it's pretty fast, only a few minutes

**Comment by hewigovens** [comment]
@maxUo I think Nick meant two parallel jobs in github actions; we can follow the same pattern in backend repo

**Comment by maxUo** [comment]
Hmm. Ok. It can be done as in the Backend repo. As I understand it, you only need help with AZP, @hewigovens ?

**Comment by EnoRage** [comment]
Waiting for @maxUo approve

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -58,3 +58,12 @@ jobs:
 
     - name: Mock Test
       run: make newman-mocked
+
+  deploy:
+    - name: CD Trigger
+      if: github.ref == 'refs/heads/master'
+      uses: Azure/pipelines@releases/v1
+      with:
+        azure-devops-project-url: 'https://dev.azure.com/TrustWallet/Trust%20BlockAtlas'
```
Comment (vikmeup) [inline .github/workflows/ci.yml:67]: change to `https://dev.azure.com/trustwallet/blockatlas`

**Pull request merged**